CONTRIBUTING.md

```diff
@@ -169,3 +170,42 @@ npm link @h5web/lib
 
 H5Web uses the [Feather icon set](https://react-icons.netlify.com/#/icons/fi).
 Icons can be imported as React components from `react-icons/fi`.
+
+## Dependencies
+
+When upgrading dependencies, take care of reading their changelogs and release
+notes. Look for potential breaking changes and for bug fixes and new features
+that may help improve the codebase.
+
+Beware that some dependencies cannot be upgraded because of issues or
+incompatibilities, while others must be upgraded by following a specific
+process. This section documents such cases:
+
+- The major version number of `@types/node` must match the version of Node
+  specified in the `engine` field of `package.json`.
+- `d3-array` (and therefore `@types/d3-array`) cannot be upgraded to v3.x due to
+  a [bug in Babel](https://github.com/babel/babel/issues/11038).
+- Dev dependencies related to ESLint must be upgraded together with
```

Comment:
I would put the list before:

> These eslint dev depedencies must be upgraded together with `eslint-config-galex:
>  - list
>  - ...
> Their **version (typo there)** numbers...
>

Reply:
Hmm, yeah but the fact that this is all inside a bullet point makes it tricky.